[2025-03-14T14:11:07.777+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-14T14:11:07.821+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_prep_transformation_model_building.data_validation manual__2025-03-14T14:10:27.322220+00:00 [queued]>
[2025-03-14T14:11:07.855+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_prep_transformation_model_building.data_validation manual__2025-03-14T14:10:27.322220+00:00 [queued]>
[2025-03-14T14:11:07.857+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-03-14T14:11:07.947+0000] {taskinstance.py:2890} INFO - Executing <Task(_PythonDecoratedOperator): data_validation> on 2025-03-14 14:10:27.322220+00:00
[2025-03-14T14:11:08.057+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=577) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-14T14:11:08.071+0000] {standard_task_runner.py:72} INFO - Started process 595 to run task
[2025-03-14T14:11:08.044+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'data_prep_transformation_model_building', 'data_validation', 'manual__2025-03-14T14:10:27.322220+00:00', '--job-id', '244', '--raw', '--subdir', 'DAGS_FOLDER/data_prep_transformation_model_building.py', '--cfg-path', '/tmp/tmpev3g4aej']
[2025-03-14T14:11:08.073+0000] {standard_task_runner.py:105} INFO - Job 244: Subtask data_validation
[2025-03-14T14:11:08.616+0000] {task_command.py:467} INFO - Running <TaskInstance: data_prep_transformation_model_building.data_validation manual__2025-03-14T14:10:27.322220+00:00 [running]> on host 0bbddc3bb474
[2025-03-14T14:11:09.230+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='data_prep_transformation_model_building' AIRFLOW_CTX_TASK_ID='data_validation' AIRFLOW_CTX_EXECUTION_DATE='2025-03-14T14:10:27.322220+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-14T14:10:27.322220+00:00'
[2025-03-14T14:11:09.248+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-14T14:11:09.617+0000] {logging_mixin.py:190} INFO - Data Validation Completed. Report saved at:
[2025-03-14T14:11:09.618+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-14T14:11:09.644+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-14T14:11:09.645+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=data_prep_transformation_model_building, task_id=data_validation, run_id=manual__2025-03-14T14:10:27.322220+00:00, execution_date=20250314T141027, start_date=20250314T141107, end_date=20250314T141109
[2025-03-14T14:11:09.824+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-14T14:11:09.897+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-14T14:11:09.914+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
